glean-core/ios/GleanTests/DispatchersTest.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  DispatchersTest.swift
+//  GleanTests
+//
+//  Created by Travis Long on 9/12/19.
+//  Copyright © 2019 Jan-Erik Rediger. All rights reserved.
+//
+
+@testable import Glean
+import XCTest
+
+class DispatchersTest: XCTestCase {
```

Comment:
Let's make extract sure this isn't flaky :) Unleash your testing obscure power :-D

Reply:
I'm pretty confident in this one, there are better ways of controlling and awaiting the async bits, but I'll try and put it through as much testing as possible.